ListOfSobjectsTest.cls

```diff
@@ -0,0 +1,28 @@
+@IsTest
+public class ListOfSobjectsTest {
+    @IsTest static void testStandardObjectsInOrg(){

```

Comment:
@isTest should be in separate line
in correct method name, format should be
testMethodName_inputdata_expectedOuput